packages/coachmark/README.md

```diff
@@ -12,30 +12,195 @@ An `<sp-coachmark>` element can be used to bring added attention to specific par
-<sp-coachmark quiet variant="dark"></sp-coachmark>
-<sp-coachmark quiet variant="light"></sp-coachmark>
+<sp-coachmark placement="right" triggerInteraction="click">
+    <sp-coachmark-popover primary-cta="Ok">
+        <div slot="title">A single coachmark</div>
+        <div slot="content">
+            This is a Coachmark with nothing but text in it. Kind of lonely in
+            here.
+        </div>
+    </sp-coachmark-popover>
+    <sp-coach-indicator slot="trigger"></sp-coach-indicator>
+</sp-coachmark>
 ```
+
+### User Action Dependent
+
+Coach marks in a tour can be dependent on a user taking an action. When coach marks are action dependent, the “next step” button should be removed since the next step will be triggered by an action. This ensures that a user can learn by taking an action, rather than just reading about it. Unless the tour is skipped, restarted, or moved back to the previous step, this will persist until the user takes a corresponding action. For example, if a coach mark is educating a user about a shared library, it will persist until the user opens the Library panel.
+
+```html
+<sp-coachmark placement="right" open>
+    <sp-coachmark-popover
+        open
+        currentstep="2"
+        totalsteps="8"
+        primary-cta="Asset added"
+        secondary-cta="Previous"
+    >
+        <div slot="title">Coachmark with user action</div>
+        <div slot="content">
+            This is a Coachmark with nothing but text in it. Kind of lonely in
+            here.
+        </div>
+        <sp-action-menu placement="bottom-end" quiet slot="actions">
+            <sp-menu-item>Skip tour</sp-menu-item>
+            <sp-menu-item>Restart tour</sp-menu-item>
+        </sp-action-menu>
+    </sp-coachmark-popover>
+    <sp-coach-indicator slot="trigger"></sp-coach-indicator>
+</sp-coachmark>
+```
+
+### with Media
+
+Coach marks can contain images or videos that relate to their content, such as demonstrations of gestures, the feature being used, or illustrations.
+
+```html
+<sp-coachmark placement="right" open>
+    <sp-coachmark-popover
+        currentstep="2"
+        totalsteps="8"
+        open
+        primary-cta="Next"
+        secondary-cta="Previous"
+        src="https://picsum.photos/id/237/200/300"
+        media-type="image"
+    >
+        <div slot="title">Coachmark with 16:9 image</div>
+        <div slot="content">This is a Coachmark with some description</div>
+        <sp-action-menu placement="bottom-end" quiet slot="actions">
+            <sp-menu-item>Skip tour</sp-menu-item>
+            <sp-menu-item>Restart tour</sp-menu-item>
+        </sp-action-menu>
+    </sp-coachmark-popover>
+    <sp-coach-indicator slot="trigger"></sp-coach-indicator>
+</sp-coachmark>
+```
+
+### with Keyboard Shortcut
+
+A coach marks can contain a keyboard shortcut that relates to its content, such as a tool or action that can be used. When the more action buttons is present, the keyboard shortcut is placed under the title.
```

Comment:
```suggestion
A coach marks can contain a keyboard shortcut that relates to its content, such as a tool or action that can be used. 
```

Could you include information about the difference between the keyboard shortcut and modifier keys in the API?